### DISCUSSION TITLE
Generating training data from trajectory

### DISCUSSION THREAD

**Discussion by jonathan-booth** [mir-group/nequip#409] [Q&A]
I have a few trajectories where, at each frame, I have a text file where each line is an atom and we have values for the atom type, the x,y,z positions, and the x,y,z components of the force vector. Each text file also has the total potential energy. I have trajectories for different sized systems. How do I convert these data into something that Nequip (Allegro really) can work with? I've read the docs but not found an answer sorry if I missed something obvious.

Thanks in advance

**Comment by Linux-cpp-lisp** [comment]
Hi @jonathan-booth ,

You need to convert it into extXYZ format; the easy way is to parse your format yourself in Python, make `ase.Atoms` objects (see their docs) and then write them out with `ase.io.write` in the extxyz format. At present only extxyz format supports variable system size inputs in our code.

**Comment by Linux-cpp-lisp** [comment]
Oh, closing as duplicate: https://github.com/mir-group/allegro/discussions/74
